cpp/serve/prefix_cache.h

```diff
@@ -0,0 +1,134 @@
+/*!
+ *  Copyright (c) 2023 by Contributors
+ * \file serve/prefix_cache.h
+ */
+#ifndef MLC_LLM_SERVE_PREFIX_CACHE_H_
+#define MLC_LLM_SERVE_PREFIX_CACHE_H_
+#include <tvm/runtime/container/shape_tuple.h>
+#include <tvm/runtime/object.h>
+#include <tvm/runtime/packed_func.h>
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include "model.h"
+#include "radix_tree.h"
+#include "request_state.h"
+
+namespace mlc {
+namespace llm {
+namespace serve {
+
+using namespace tvm::runtime;
+
+/*!
+ * \brief The matched result from prefix cache.
+ */
+struct MatchedResult {
```

Comment:
PrefixCacheMatchedResult, given this namespace is serve and MatchedResult is a bit generic

Reply:
renamed